orte/orted/orted_main.c

```diff
@@ -747,7 +756,7 @@ int orte_daemon(int argc, char *argv[])
 
         /* define the target jobid */
         target.jobid = ORTE_PROC_MY_NAME->jobid;
-        if (orte_fwd_mpirun_port || orte_static_ports || NULL != orte_parent_uri) {
+        if (orte_fwd_mpirun_port || NULL != orte_parent_uri) {
```

Comment:
I believe you have to remove the `orte_fwd_mpirun_port` here as well since we now need everyone to follow this procedure as we cannot compute the connection map in advance.

Reply:
will be in the next commit.